packages/plugin-legacy/src/types.ts

```diff
@@ -3,6 +3,10 @@ export interface Options {
    * default: 'defaults'
    */
   targets?: string | string[] | { [key: string]: string }
+  /**
+   * default: ['es2020', 'edge79', 'firefox67', 'chrome64', 'safari12']
+   */
+  modernTargets?: string | string[]
```

Comment:
Shouldn't `modernTargets` be accepting browserslist syntax so it's consistent with the `targets` option? And we'd need to pass that to here:

https://github.com/vitejs/vite/blob/b44c49302ffbf0c82f984f6219ed6376d1e4552a/packages/plugin-legacy/src/index.ts#L397

Then, we also need to convert the `modernTargets` browserslist syntax to esbuild using https://github.com/nihalgonsalves/esbuild-plugin-browserslist